surface-model/surface-model_shacl.ttl

```diff
@@ -0,0 +1,141 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix surface-model: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/surface-model/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix general: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/> .
+@prefix georeference: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/georeference/> .
+
+surface-model:surfaceModelShape a sh:NodeShape ;
+    sh:property [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node general:GeneralShape ;
+            sh:order 1 ;
+            sh:path surface-model:general ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node surface-model:FormatShape ;
+            sh:order 2 ;
+            sh:path surface-model:Format ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node surface-model:DataShape ;
+            sh:order 3 ;
+            sh:path surface-model:Data ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node surface-model:QualityShape ;
+            sh:order 4 ;
+            sh:path surface-model:Quality ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node surface-model:QuantityShape ;
+            sh:order 5 ;
+            sh:path surface-model:Quantity ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node georeference:GeoreferenceShape ;
+            sh:order 6 ;
+            sh:path surface-model:Georeference ] ;
+    sh:targetClass surface-model:surface-model .
```

Comment:
When ontology is fixed it should be 
`sh:targetClass surface-model:SurfaceModel .`

Reply:
fixed